pages/en/deploying/subgraph-studio-faqs.mdx

```diff
@@ -2,25 +2,29 @@
 title: Subgraph Studio FAQs
 ---
 
-### 1. How do I create an API Key?
+### 1. What is Subgraph Studio?
+
+Subgraph Studio is a platform for creating, managing, and publishing subgraphs and API keys.
```

Comment:
Is "platform" the right term? What about "web app" or "dapp"? Also, should we link to it (https://thegraph.com/studio/)? Seems odd not to.

Reply:
I think dapp works best! Good call.

I'll add that link in too. Thanks!